### PR TITLE
Added option to clear cache by path

### PR DESCRIPTION
**Pull request by jacknm**
This PR adds an option to clear cache by path. It also updates the clear cache page description text to make it easier to understand the differences between the available options.

Default View
default

After Cache Cleared
after clear cache

### CODE REVIEW

File: inc/tpl/firewall-clearcache.html.tpl
Comment (bfeddern-godaddy) [inline inc/tpl/firewall-clearcache.html.tpl:46]: You've got me reviewing... _double-checks_... jQuery?  WHAT YEAR IS IT?!?!  😜

File: inc/tpl/firewall-clearcache.html.tpl
Comment (bfeddern-godaddy) [inline inc/tpl/firewall-clearcache.html.tpl:27]: is that... a `var` usage?  see comment where i ask what year this is.  i think i woke up in the early 2000s.

File: inc/tpl/firewall-clearcache.html.tpl
Comment (bfeddern-godaddy) [inline inc/tpl/firewall-clearcache.html.tpl:105]: see, i told you that you knew english better than me!

File: src/firewall.lib.php
Comment (bfeddern-godaddy) [inline src/firewall.lib.php:668]: should we maybe adjust the spacing on this variable declaration to be consistent?

**Review by bfeddern-godaddy — APPROVED** [review]
Looks good to me, but might be wise to wait for a review from @gvre before merging and deploying.

File: inc/tpl/firewall-clearcache.html.tpl
Comment (jacknm) [reply inc/tpl/firewall-clearcache.html.tpl:105]: He he all credits go to Rianna :)

**Review by gvre — APPROVED** [review]
GJ 👍

**Review by gvre — APPROVED** [review]

**Pull request merged**
